src/expr/udf/src/external.rs

```diff
@@ -196,6 +200,7 @@ impl ArrowFlightUdfClient {
                     return ret;
                 }
             }
+            metrics.udf_failure_count.with_label_values(labels).inc();
```

Comment:
Hmm should we use `udf_failure_count` to track this as well? Or perhaps a new metric?

Because `udf_failure_count` implies that the function has failed. Whereas we here we are tracking `udf_retry_count` instead.

Reply:
Good suggestion. I already add a `udf_retry_count` metrics.